sources/Makefile.am

```diff
@@ -13,9 +13,9 @@ sources =
 
 build_sources = AssemblyInfo.cs $(sources)
 
-CLEANFILES = $(DLL) generated-stamp generated/*.cs $(API)
+CLEANFILES = $(DLL) generated-stamp generated/* $(API)
 
-DISTCLEANFILES = AssemblyInfo.cs $(DLLMAP)
+DISTCLEANFILES = AssemblyInfo.cs
```

Comment:
The .dll.config needs to be installed with the .dll file. Isn't ```EXTRA_DIST``` the place to put those files?